### PR TITLE
PXP-426: [CLI] Show which build artifact is the current deployed one when selecting artifact

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

Ticket: [PXP-426](https://mindvalley.atlassian.net/browse/PXP-426)

* This is using this [PR](https://github.com/mindvalley/wukong-api-proxy/pull/47) as backend to get the `buildBranch` value

## What's Changed

- [x] show the current build on the selection in `wukong deployment execute` command
    - if the build selection list contains the current build, it will display in 3 columns layout
      Screenshot 2022-12-06 at 4 17 13 PM
    - if the current `build_artifact` is not in `{branch}-build-{build_number}` pattern, or the build selection list doesn't contain the current build, it will display in 2 columns layout
      Screenshot 2022-12-06 at 4 37 49 PM

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/46#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/46#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/46#pullrequestreview-1213033356" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
